Framework/PythonInterface/plugins/algorithms/IntegratePeaksProfileFitting.py

```diff
@@ -193,6 +200,7 @@ def PyExec(self):
                     numgood += 1
 
             except KeyboardInterrupt:
+                np.warnings.filterwarnings('default') # Re-enable on exit
```

Comment:
I think this is in the wrong place. Shouldn't it go in a `finally` clause of the `try`/`except`?

Reply:
This just makes sure warnings are turned back on if the algorithm is terminated early.  If it goes in a `finally` clause then I'll need to turn off warnings again at the start of each peak, which seems unnecessary.  If the algorithm finished normally, warnings are turned back on right before exiting (line 215).

If it's still best practice to put it in a `finally` statement let me know and I can change it.